### PR TITLE
Use array delete instead of scalar delete.

### PR DESCRIPTION
**Pull request by won3d**
I also tried setting the node version to `>= 8.10` and that seemed to work fine. Should I commit `package.json` too?

### CODE REVIEW

File: src/webgl.cc [hunk 1]
```diff
@@ -1764,11 +1764,11 @@ NAN_METHOD(GetExtension) {
   char *sname=*name;
   char *extensions=(char*) glGetString(GL_EXTENSIONS);
   char *ext=strcasestr(extensions, sname);
-  
-  if(ext==NULL){ 
+
+  if(ext==NULL){
```
Comment (hamham91) [inline src/webgl.cc]: please just run clang-format on this file ☠️ ☠️ ☠️

File: src/webgl.cc [hunk 2]
```diff
@@ -1384,7 +1384,7 @@ NAN_METHOD(GetShaderSource) {
   glGetShaderSource(shader, len, NULL, source);
 
   Local<String> str = JS_STR(source);
-  delete source;
+  delete [] source;
```
Comment (hamham91) [inline src/webgl.cc:1387]: what made you notice this, a memory leak?

**Review by hamham91 — COMMENTED** [review]
👍 lgtm

yea, commit the package.json

File: src/webgl.cc (hunk 2 above)
Comment (won3d) [reply src/webgl.cc:1387]: I read the clang warnings. 🙃

File: src/webgl.cc (hunk 1 above)
Comment (won3d) [reply src/webgl.cc]: Ugh, I was thinking of the opposite: backing off the whitespace changes. Probably want a minimal change in case we want to upstream?

Comment (emma-floored) [reply src/webgl.cc]: I agree - in this case might it make sense to drop the whitespace changes

**Comment by won3d** [comment]
PTAL

I also added another quick fix for the deprecation warning spam, so currently node-webgl builds cleanly. There are some warnings in node-glfw but meh.

BTW, I tested this by going to my `depot` repo and doing `npm install ../node-webgl` to test. Is this the right/best way?

**Comment by emma-floored** [comment]
👍

**Pull request merged**